sonic-ledd/scripts/ledd

```diff
@@ -124,10 +275,11 @@ def main():
 
     ledd = DaemonLedd()
 
-    # Listen indefinitely for changes to the PORT table in the Application DB's
+    # Listen indefinitely for port oper status changes
     while True:
-        ledd.run()
-
+        if 0 != ledd.run():
+            print("ledd.run() failed... Exiting")
```

Comment:
add/modify this  a syslog here please

Reply:
@vdahiya12 I just kept the behavior as old one. if you see the main() function, it has many other prints....these exit log do come to syslog ?

Reply:
we will catch it in supervisor logs, the print logs can remain. Ideally, we should have disallowed print statements as is, they won't go to syslog as I recall, but supervisor logs will if the daemon hangs/exits. For now we can keep as is.